### ISSUE TITLE
Z image training

### ISSUE THREAD

**Issue by spike4379** [tdrussell/diffusion-pipe#473]
Hey so ive seen people running lora training on this and theres no example training tomls or settings required for training?
How are people knowing what to even do with the training and parameters?

**Comment by LorisTecnology** [comment]
i did tried to train on Z image and got this as result

<img width="1322" height="196" alt="Image" src="https://github.com/user-attachments/assets/4aeb6239-6ade-48bd-bf38-1ed5ef1a1262" />

1KB files

**Comment by Jamzsam** [comment]
These are my toml configs. Any more than 30 - 40 images tend to take too long and over fitting occurs. I had ok results. Lowest loss LR i could get was 0.7400. Still probably best wait for the Z Image Base model to come out in my opinion.

Z_image_24gb_vram2.toml

output_dir = "/mnt/b/Output"
dataset = "/home/Anon/diffusion-pipe12/Z_image_24gb_vram_dataset.toml"

epochs = 80                 # usually enough (200–400 steps total)
micro_batch_size_per_gpu = 1
gradient_accumulation_steps = 1
gradient_clipping = 1
warmup_steps = 20

eval_every_n_epochs = 1
eval_before_first_step = true
save_every_n_epochs = 1
checkpoint_every_n_minutes = 60
activation_checkpointing = true
partition_method = "parameters"
save_dtype = "bfloat16"
caching_batch_size = 8
steps_per_print = 10

[model]
type = "z_image"
diffusion_model = "/mnt/a/Comfy WSL-NEW4/models/diffusion_models/z_image_turbo_bf16.safetensors"
vae = "/mnt/a/Comfy WSL-NEW4/models/vae/ae.safetensors"
text_encoders = [{ path = "/mnt/a/Comfy WSL-NEW4/models/text_encoders/qwen_3_4b.safetensors", type = "lumina2" }]
merge_adapters = ["/mnt/a/Comfy WSL-NEW4/models/loras/Z-Image/Z-ImageTrainingAdapter/zimage_turbo_training_adapter_v2.safetensors"]
dtype = "bfloat16"
#transformer_dtype = 'float8'
#timestep_sample_method = 'logit_normal'**

[adapter]
type = "lora"
#rank = 32
rank = 64
dtype = "bfloat16"

[optimizer]
type = "adamw"
#lr = 8e-5
lr = 1e-4                 # 4–10× higher than Flux — this is the speed secret
betas = [0.9, 0.99]
#weight_decay = 0.01
weight_decay = 0.0001
eps = 1e-8

-----------------------------------------------------------------------------------------------
Z_image_24gb_vram_dataset.toml

# Resolution settings.

resolutions = [[1024,1024]]

# Aspect ratio bucketing settings
enable_ar_bucket = true
min_ar = 0.5
max_ar = 2.0
num_ar_buckets = 9

# Frame buckets (1 is for images)
frame_buckets = [1]
#video_clip_mode = 'single_beginning'

[[directory]]
# Set this to where your dataset is
path = '/mnt/g/Training/Images'
#mask_path = '/mnt/g/Training/MaskImages'
# Reduce as necessary
num_repeats = 1

**Comment by LorisTecnology** [comment]
thanks a lot. I'm testin now with adamw_optimi and seems to be really fast
5 seconds per Sample
